nodeoperators/nodelist.json

```diff
@@ -2881,18 +2881,42 @@
     },
     {
         "Role": "access",
-        "Address": "192.168.56.114:3569",
-        "NodeID": "aa597fced846dcd66c886f53b0dfdc1417e4eac351ff7091369afa5ab88ed8af",
-        "Stake": 1000,
-        "NetworkPubKey": "7115c4ca60edc8e1c4ea250aa5670f503a9f24591cd0a18ad35695d1271efc4f8004a0f0101a89d8659bbf0cf10249b3b5fb59d4a266658a51cb54cc213356bd",
-        "StakingPubKey": "9693bab0f6b1f2eac44d9b1c745462b25c652a97476f03f4d6e72139bd48e7881c7f734e61a82a03144984cf5c2c6f8e196901a60c903c86fd630c2010d5522c30aa1f879ef99acc94fec1b5e2215845d67951a8e63c30d9677918d06655944f"
-    },
-    {
-        "Role": "access",
-        "Address": "172.20.4.50:3569",
-        "NodeID": "c0b6fa1ab45151573e2071469a92b1cae39b4f154232fdfed3884fb9a88f14fc",
-        "Stake": 1000,
-        "NetworkPubKey": "b98684995a9f363bec5214ad159e56d78665321b3cc56fec5a3b136de8f2a87ba37f81dc3516d394e6082717254535dd11e7126eeec21b78ba7c03fb9c3460af",
-        "StakingPubKey": "ae1e3fce300ca4fcc6c67975bf37b0736ddca2361b33d7f78fc9b9377997f255f252945cab2adbce09587c0846206c95081164b4f5b15ed8589b810dcc88ac50de29039e35b10c6a3bb076e85b4314e9a2b62f8937eacf15304bd85f3e6f8458"
+        "Address": "3.35.44.32:3569",
+        "NodeID": "55501da7d7db34bd63d27f97a6c8f73c6372958f77f3292cb00eb12555737061",
+        "Stake": 1000,
+        "NetworkPubKey": "0756d871636d826cd83217404510609b81d0bed8669a2e7f7b5adb3659a8a2e564bf7f567c270b2d845c43b65d56ac43b8672beaf898a96a511ea0ef7a0add6f",
+        "StakingPubKey": "b3d1071a1d49be3cd84672e04677499d8a8b7dcfd4a7d18e3821e367beabb210eb84dd9de1c1973233dd72a22394365201a5d579a65baac9bf7aec588866740b11aa452c19cf7f441bc06f93fca949d36cb5cc3d2cc4068eba37021b0eaad382"
+    },
+     {
+      "Role": "access",
+      "Address": "18.178.119.191:3569",
```

Comment:
Coinone Access node 2

Reply:
is this coinone access node 1 or 2?

Reply:
My comments were incorrect (or they moved after I made the second commit), not sure. Deleting and adding comments again.